config/demo.exs

```diff
@@ -0,0 +1,27 @@
+import Config
+
+config :wanda, Wanda.Policy, execution_server_impl: Wanda.Executions.FakeServer
+
+config :wanda, WandaWeb.Endpoint,
+  check_origin: :conn,
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  server: true
+
+config :logger, level: :debug
+
+config :wanda, Wanda.Messaging.Adapters.AMQP,
+  consumer: [
+    connection: "amqp://wanda:wanda@localhost:5672"
+  ],
+  publisher: [
+    connection: "amqp://wanda:wanda@localhost:5672"
+  ],
+  processor: Wanda.Messaging.Adapters.AMQP.Processor
+
+config :cors_plug,
```

Comment:
I don't think we need this `cors_plug`. We can use the same as in config, and simply set some env variable to `CORS_ORIGIN`. It is even changed in runtime

Reply:
CORS_ORIGIN wasn't enough in this case, I had a conversation with @CDimonaco about it. Apparently we don't have this happening in prod as in our helm chart everything comes from the same domain.

Reply:
But you could still use the `CORS_ORIGIN` and only leave the `allow_crendentials` value (even though I don't find this fields documented anywhere nor in the code).
I think having `http://demo.trento-project.io` should be enough, as the https is not even available in our demo

Reply:
I think this `allow_credentials` usage is a mistake and it is coming from other package usage, [corsica](https://github.com/whatyouhide/corsica)

Reply:
Fair enough, I'm currently working on building a demo container in the CI for deployment in our demo env. I'll revisit this issue and remove this if I can by using env vars where possible.

Reply:
I've removed it for now as it's only needed for development. The demo environment when deployed through helm doesn't need it.